packages/expo-barcode-scanner/android/build.gradle

```diff
@@ -81,10 +81,7 @@ android {
 
 dependencies {
   implementation project(':expo-modules-core')
-
-  api 'com.google.android.gms:play-services-vision:19.0.0'
-  api 'com.google.zxing:core:3.3.3'
-
+  api 'com.google.mlkit:barcode-scanning:17.1.0'
```

Comment:
this adds `libbarhopper_v3.so` which dramatically increases the build size
see: https://github.com/expo/expo/issues/26749